Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -559,6 +559,8 @@ public void OnPointerDown(MixedRealityPointerEventData eventData)
             if (eventData.used ||
                     (!allowFarManipulation && eventData.Pointer as IMixedRealityNearPointer == null))
             {
+                Debug.LogError("AllowFarManipulation is false and a NearInteractionGrabbable component is not attached to this object. " +
```

Comment:
Is this the only way to get into this `if` case? I don't see where we're checking for `NearInteractionGrabbable` existing to get to this point, just checking if the pointer is a far or near pointer.

Reply:
This is not the only way, this error message is logged when there is far interaction input on the object:
![ObjectManipulatorErrorMessage](https://user-images.githubusercontent.com/53493796/105079366-2af66980-5a44-11eb-8788-38fbcb58fae5.gif)

But I think it might be better to add a clearer case in start.

```
        private void Start()
        {
            rigidBody = HostTransform.GetComponent<Rigidbody>();
            if (constraintsManager == null && EnableConstraints)
            {
                constraintsManager = gameObject.EnsureComponent<ConstraintManager>();
            }

            if (!AllowFarManipulation && GetComponent<NearInteractionGrabbable>() == null)
            {
                Debug.LogError($"AllowFarManipulation is false and a NearInteractionGrabbable component is not attached to {gameObject.name}. " +
                " To enable near interaction support, attach a NearInteractionGrabbable component to this object.");
            }
        }
```

Reply:
Moved error message